src/main/java/com/iexec/core/replicate/ReplicatesService.java

```diff
@@ -170,6 +169,16 @@ private int getNbReplicatesWithGivenStatusJustBeforeWorkerLost(String chainTaskI
         return Optional.empty();
     }
 
+    public Optional<Replicate> getReplicateWithResultUploadedStatus(String chainTaskId) {
```

Comment:
It seems we have an issue when worker is lost after UPLOADED

Reply:
https://github.com/iExecBlockchainComputing/iexec-core/blob/301656cd5c2353d5c131c95c330717bba388715a/src/main/java/com/iexec/core/replicate/ReplicatesService.java#L148

Reply:
great catch, thanks! I've modified the code.